src/main/java/gov/nist/csd/pm/pap/ProhibitionsAdmin.java

```diff
@@ -10,7 +12,10 @@
 
     private Prohibitions prohibitions;
 
-    public ProhibitionsAdmin(Prohibitions prohibitions) {
+    public ProhibitionsAdmin(Prohibitions prohibitions) throws PMException {
+        if (!(prohibitions instanceof MemProhibitions)) {
+            this.prohibitions = new MemDBProhibitions(prohibitions);
+        }
```

Comment:
same thing here about removing the if statement and just passing MemDBProhibitions as the Prohibitions parameter when using it (i.e. the admin tool)